next-tavla/app/(admin)/organizations/components/MemberAdministration/index.tsx

```diff
@@ -3,23 +3,25 @@ import { MemberList } from './MemberList'
 import { TOrganizationID, TUser, TUserID } from 'types/settings'
 import { InviteUser } from './InviteUser'
 import { Contrast } from 'Admin/components/Contrast'
-
+import styles from './CountiesSelect/styles.module.css'
```

Comment:
Legg det inn i en egen styles fil, vi pleier å kalle styles vi importer for `classes`